torch/ao/quantization/backend_config/tensorrt.py

```diff
@@ -5,6 +5,7 @@
 import torch.nn.intrinsic.qat as nniqat
 # TODO: maybe refactor this to a separate util function
 from .native import _get_binary_op_configs
+from .native import _get_share_qparams_op_configs
```

Comment:
what does "native" mean? is it fbgemm/qnnpack?  If yes, does it make sense to import from there?

Reply:
yeah it is fbgemm/qnnpack, I have a TODO to move this to a util function above to clean up this. or would you like to do this now?

Reply:
a separate PR is ok, but the current code is a bit confusing